src/server/serve.rs

```diff
@@ -313,18 +366,21 @@ impl InnerService {
 
         // Prepare response body.
         // Being mutable for further modifications.
-        let mut body: io::Result<Vec<u8>> = Ok(Vec::new());
+        let mut body = Body::empty();
+        let mut content_length: Option<u64> = None;
```

Comment:
Type annotation is optional here.

```suggestion
        let mut content_length = None;
```

Reply:
applied on 481b1fb